.github/workflows/test.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         jvm: ["8"]
         scala: ["2.13.10", "2.12.17"]
         espresso: ["2.4"]
-        circt: ["sifive/1/21/0"]
```

Comment:
Does 1/24/0 work?

Reply:
probably, but we can do that in a follow-up to keep this tidy?